updatecli/updatecli.d/debian.yaml

```
@@ -49,8 +49,8 @@ actions:
   default:
     kind: github/pullrequest
     scmid: default
-    title: Bump Debian Bookworm Linux version to {{ source "bookwormLatestVersion" }}
+    title: Bump Debian trixie Linux version to {{ source "trixieLatestVersion" }}
     spec:
       labels:
         - dependencies
-        - debian-bookworm
+        - debian-trixie
```

Let's simplify this and keep only "debian", having the release line here doesn't add any value
```suggestion
        - debian
```